staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go

```diff
@@ -95,6 +95,26 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
```

Comment:
This isn't the place where those metrics should live.
We have metrics in cacher.go already:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go#L57

Those should be added to watchcache.go

Alternatively (or even better) would be if you could open a separate PR, where you create metrics.go file in cacher/ pkg and move the metrics from cacher.go there. And in this PR you will extend that file with these metrics.

Reply:
Sure, I'm glad to.

Reply:
Please assign it directly to me once you open it .

Reply:
OK, I think it's a very recent work.